internal/publish/model/exposure_model.go

```diff
@@ -481,11 +481,11 @@ func (t *Transformer) TransformPublish(ctx context.Context, inData *verifyapi.Pu
 		if onsetInterval > 0 {
 			daysSince := DaysFromSymptomOnset(onsetInterval, exposure.IntervalNumber)
 			if abs := math.Abs(float64(daysSince)); abs > t.maxSymptomOnsetDays {
-				logger.Debugw("dropping key due to symptom onset magnitude too high", "daysSince", daysSince)
```

Comment:
Can you comment why (specifically that some states had hard-coded this to 1)? I want to avoid accidentially removing it.

Reply:
added